examples/todo/imports/ui/profile/ProfileContainer.js

```diff
@@ -19,11 +21,58 @@ export const editProfileMutation = graphql(
   }
 );
 
+
 const loggedUser = getLoggedUserContext();
 
+handleSubmitProfile = (editProfileState) => !!editProfileState.firstName.trim()
+
 export const ProfileContainer = compose(
   editProfileMutation,
   loggedUser,
   withRouter,
-  withApollo
+  withApollo,
+  // withAlert,
+  withState('editProfileState', 'setEditProfileState', props =>
+    intialState = {
+      _id: props.loggedUser._id,
```

Comment:
why not only ({ ....  })

you are not declaring intialState (typo anyway, should be initialState) and not using it, just implicit returning it

Reply:
Yes, I changed for it, you can see at: https://github.com/CodeFTW/future-web/commit/bc1b027604ef8d8870f7b64a267a671561bb64cd
Thanks